packages/pwa-kit-react-sdk/bin/upload.js

```diff
@@ -42,38 +44,59 @@ const argv = require('yargs')
         describe: 'a custom target to upload a bundle to within Mobify Cloud',
         type: 'string'
     })
+    .option(POTENTIAL_OPTIONS.config, {
+        alias: 'config',
+        describe:
+            'relative path to the configuration file, default to the mobify key in package.json',
+        type: 'string'
+    })
     .help('h')
     .alias('h', 'help')
     .strict().argv
 
-let packageJson = {}
-try {
-    // Assumption: this binary is invoked from the project root directory
-    packageJson = JSON.parse(fs.readFileSync('package.json', {encoding: 'utf8'}))
-} catch (e) {
-    let message
-    if (e.code === 'ENOENT' && e.path === 'package.json') {
-        message =
-            '[Error: package.json not found. Are you running this command in the project root directory?]' // eslint-disable-line max-len
-    } else {
-        message = e.message
+const getOptions = (configFilePath) => {
+    console.log('getoptions')
+    if (configFilePath) {
+        return getOptionsFromConfig(configFilePath)
     }
+    return getOptionsFromPackageJson()
+}
+
+const getOptionsFromConfig = (configFilePath) => {
+    const config = require(path.resolve(process.cwd(), configFilePath))
```

Comment:
I am having trouble importing ES module from this commonjs module. This is one of the downside of using `.js` config file format. I am trying to resolve this.